sdk/attestation/attestation/src/attestationClient.ts

```diff
@@ -99,17 +142,114 @@ export class AttestationClient {
     // Legacy compatibility classes functions which will be removed eventually.
     this.policy = new Policy(this);
     this.policyCertificates = new PolicyCertificates(this);
-    this.attestation = new Attestation(this);
   }
 
   /**
+   * @ignore
+   * @hidden
+   * 
    * Temporary function to access the generated client, used for the operations
    * TS files.
    * @returns The generated client for the attestation service.
    */
-  public BaseClient() : GeneratedClient
+   public BaseClient() : GeneratedClient
+   {
+     return this._client;
+   }
+
+  /** Attests an OpenEnclave report generated from an SGX Enclave using the OpenEnclave SDK.
+   * 
+   * @param report An OpenEnclave report generated by an SGX enclave.
+   * @param options Operation options for the attestOpenEnclave API call.
+   * @returns 
+   */
+  public async attestOpenEnclave(report : Uint8Array, options : AttestOpenEnclaveOptions={}) : Promise<AttestationResponse<AttestationResult>> {
+    const { span, updatedOptions} = createSpan("AttestationClient-attestOpenEnclave", options);
+    try {
+      const attestationResult = await this._client.attestation.attestOpenEnclave({
+        report: report,
+        initTimeData: (options.initTimeData ? { 
+          data: options.initTimeData.data,
+          dataType: (options.initTimeData.isJson ? "JSON" : "Binary"),
+        } : undefined),
+        runtimeData:  (options.runTimeData ? { 
+          data: options.runTimeData.data,
+          dataType: (options.runTimeData.isJson ? "JSON" : "Binary"),
+        } : undefined),
+        draftPolicyForAttestation: (options.draftPolicyForAttestation ?? undefined),
+      }, updatedOptions);
+
+      let token = new AttestationToken(attestationResult.token ?? "");
```

Comment:
What will happen if the token is an empty string? If it is required by the service, let's make it required here as well.

Reply:
Updated the swagger in the readme.md.